static/app/actionCreators/dashboards.tsx

```diff
@@ -95,7 +98,8 @@ export function updateDashboard(
     const errorResponse = response?.responseJSON ?? null;
 
     if (errorResponse) {
-      addErrorMessage(errorResponse);
+      const errors = flattenErrors(errorResponse, {});
```

Comment:
should we maybe use the function `handleXhrErrorResponse`? maybe we could improve it to support all use cases

Reply:
The flatten function just parses nested error responses whereas the handleXhrErrorResponse logs stuff to Sentry? I think these are all handled errors being returned here that don't need to be logged to Sentry